README.md

```diff
@@ -108,6 +110,9 @@ require("onenord").setup({
   custom_highlights = {
     TSConstructor = { fg = colors.dark_blue },
   },
+  custom_styles = {
+    LspDiagnosticsUnderlineError = "undercurl",
```

Comment:
Hi @rmehri01 , doesn't `custom_highlights` also support this like this? 🤔 
```lua
custom_highlights = {
  LspDiagnosticsUnderlineError = { style = 'undercurl' },
},
```

Reply:
Oh, I think you're right. Whoops, I thought that `custom_highlights` overwrote the whole group. Thanks!